app/controllers/qa_controller.rb

```diff
@@ -93,7 +96,16 @@ def facet_grouped_allowed_values
     @facet_grouped_allowed_values ||= current_facet["facet"]["allowed_values"].group_by { |filter| filter["value"] }
   end
 
+  def custom_options?
```

Comment:
I wonder if this check is working? The page is no longer pulling out the custom label.
See: http://finder-frontend-pr-1228.herokuapp.com/prepare-business-uk-leaving-eu?business_activity-yesno=yes&sector_business_area%5B%5D=electronics-parts-machinery&page=3